.github/workflows/lint.yaml

```diff
@@ -24,6 +24,17 @@ jobs:
         run: pip install -r ${GITHUB_WORKSPACE}/requirements.txt
       - name: Download Resources
         run: python ${GITHUB_WORKSPACE}/download_resources.py
+      - name: Download GN via CIPD
+        env:
+          GN_SHA256SUM: 'af7b2dcb3905bca56655e12131b365f1cba8e159db80d2022330c4f522fab2ef  /tmp/gn.zip'
+          GN_HASH: r3styzkFvKVmVeEhMbNl8cuo4VnbgNICIzDE9SL6su8C
+        run: |
+          set -x
+          set +e
```

Comment:
Wouldn't setting bash option to +e cause failures in the bash script to be ignored?

Reply:
Good catch, I copied the wrong command